### PR TITLE
Add a field called integration_name under TendrlContext

### PR DESCRIPTION
**Pull request by nnDarshan**
This patch adds a new attr under TendrlContext called
integration_name which indicates the name of the underlying
ceph cluster.

tendrl-bug-id: Tendrl/ceph-integration#126
Signed-off-by: nnDarshan <darshan.n.2024@gmail.com>

### CODE REVIEW

**Comment by nnDarshan** [comment]
@r0h4n @nthomas-redhat @anivargi Please Review

File: tendrl/ceph_integration/objects/tendrl_context/__init__.py [hunk 1]
```diff
@@ -20,8 +20,25 @@ def __init__(self, integration_id=None, fsid=None, *args, **kwargs):
...
         self._etcd_cls = _TendrlContextEtcd
 
+    def _get_integration_name(self):
+        cluster_name = ""
+        # TODO(team) This file is valid for CentOS and RHEL. Needs to
+        # be handled while ubuntu support is needed.
+        ceph_cfg_file = "/etc/sysconfig/ceph"
+        if not os.path.exists(ceph_cfg_file):
+            LOG.warning("config file: %s not found" % ceph_cfg_file)
+            return cluster_name
```
Comment (r0h4n) [inline tendrl/ceph_integration/objects/tendrl_context/__init__.py:34]: @nthomas-redhat what is the default ceph cluster name if none is supplied during cluster creation? We should return that default name here

File: tendrl/ceph_integration/objects/tendrl_context/__init__.py [hunk 2]
```diff
@@ -20,8 +20,25 @@ def __init__(self, integration_id=None, fsid=None, *args, **kwargs):
...
+
+        with open(ceph_cfg_file) as f:
+            for line in f:
+                if line.startswith("CLUSTER="):
+                    cluster_name = line.split('\n')[0].split('=')[1]
```
Comment (r0h4n) [inline tendrl/ceph_integration/objects/tendrl_context/__init__.py]: use line.split('\n')[0].split('=')**[-1]**  if the cluster_name is the last item

**Review by r0h4n — CHANGES_REQUESTED** [review]

File: tendrl/ceph_integration/objects/tendrl_context/__init__.py (hunk 2 above)
Comment (nnDarshan) [reply tendrl/ceph_integration/objects/tendrl_context/__init__.py]: ack

File: tendrl/ceph_integration/objects/tendrl_context/__init__.py (hunk 1 above)
Comment (nnDarshan) [reply tendrl/ceph_integration/objects/tendrl_context/__init__.py:34]: Its "ceph" AFAIK. @nthomas-redhat Please confirm

Comment (r0h4n) [reply tendrl/ceph_integration/objects/tendrl_context/__init__.py:34]: Then "ceph" should be the default cluster_name

Comment (nnDarshan) [reply tendrl/ceph_integration/objects/tendrl_context/__init__.py:34]: ack

**Review by r0h4n — APPROVED** [review]

**Pull request merged**